sentry_sdk/integrations/django/__init__.py

```diff
@@ -346,6 +346,31 @@ def _before_get_response(request):
         )
 
 
+def _after_get_response(request):
+    # type: (WSGIRequest) -> None
+    """
+    Some django middlewares overwrite request.urlconf
+    so we need to respect that contract,
+    so we try to resolve the url again.
+    """
+    if not hasattr(request, "urlconf"):
+        return
+
+    hub = Hub.current
+    integration = hub.get_integration(DjangoIntegration)
+    if integration is None or integration.transaction_style != "url":
+        return
+
+    with hub.configure_scope() as scope:
+        try:
+            scope.transaction = LEGACY_RESOLVER.resolve(
+                request.path_info,
+                urlconf=request.urlconf,
+            )
+        except Exception:
+            pass
```

Comment:
This is probably because integrations (and sdks in general) should fail as silent as possible right?